package.json

```diff
@@ -13,6 +13,7 @@
     "autoprefixer": "6.3.6",
     "browser-sync": "2.12.8",
     "cssnano": "3.5.2",
+    "fs-extra": "^0.30.0",
```

Comment:
Added the strict dependency on fs-extra. Might be good to enable https://greenkeeper.io on the package if we're doing this. Will create a seperate issue for that 😉